Week3/homework/step2-3.js

```diff
@@ -2,39 +2,44 @@
 
 // Use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // eslint-disable-next-line prefer-const
-  let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
-  return result;
+  // repeat after me
+  let repeatString = '';
+
+  for (; num > 0; ) {
```

Comment:
This works, but I'm not sure where you learnt it from! I'd prefer to see properly written for loops, with the variable declaration and the incrementer